### DISCUSSION TITLE
Welcome to hande Discussions!

### DISCUSSION THREAD

**Discussion by ran123123123** [ran123123123/hande#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MackDing** [comment]
asdasd

**Reply by ran123123123** [reply]
good@

**Comment by MackDing** [comment]
# python下安装包 墙烈推荐百度云镜像
pip3 install opencv-python-headless<4.6 -i https://mirror.baidu.com/pypi/simple
pip install -r .\help\requirement.txt -i http://pypi.douban.com/simple --trusted-host pypi.douban.com

# 全局配置pip
pip install --upgrade pip
pip config set global.index-url https://mirror.baidu.com/pypi/simple
# or
python -m pip config set global.index-url https://mirror.baidu.com/pypi/simple
python -m pip config list

**Reply by ran123123123** [reply]
# 判断当前环境是否在容器
cat /proc/1/cgroup

# 查看最10条登录ip 以及历史命令执行日期
last -10 
last -10f /var/log/wtmp

# 查看端口被哪个进程占用
lsof -i :8083

# 查看/var/log/secure文件寻找可疑IP登陆次数
cat /var/log/secure|grep 192.168.56.51

# 让history 命令显示日期和时间
echo 'HISTTIMEFORMAT="%F %T "' >> /etc/profile
source /etc/profile

# shell时间戳
date "+%Y_%m_%d-%H:%M:%S"
echo `date "+%Y_%m_%d-%H_%M_%S"`

# 显示登录系统的用户
who -Hu

**Comment by ran123123123** [comment]
tkkkkk
